src/app/store/selectors/VehicleTestResultModel.selectors.ts

```diff
@@ -9,3 +11,26 @@ export const selectSelectedVehicleTestResultModel = createSelector(
   selectFeature,
   (state: IVehicleTestResultModelState) => state.vehicleTestResultModel
 );
+
+export const selectTestTypeById = (id: string) => createSelector(
+  selectFeature,
+  (state: IVehicleTestResultModelState) => {
+    let testType: TestType;
+    let testRecord: TestResultModel;
+
+    if (state.vehicleTestResultModel) {
```

Comment:
This will need a unit test for that.
We can extract that into a separate function if it makes it hard to unit test that we then use in this selector.
It seems like it is taking a vehicleTestResultModel as input and testType.

Reply:
This selector has been simplified in the implementation of 10278